pyproject.toml

```diff
@@ -30,6 +31,8 @@ dependencies = [
     "numpy",
     "necessary>=0.4.3",
     "charset-normalizer>=3.2.0",
+    "zstandard>=0.20.0",
+    "backoff>=2.0.0",
```

Comment:
Is this version required? There's 2 minor versions since this 2.0 release "2.2.1"